freedomofpress/securedrop
# Apparmor: allow apache2 to execute /usr/bin/file

## Status

Ready for review

## Description of Changes

Fixes #3020.
AppArmor profile now allows `/usr/sbin/apache2` to execute `/usr/bin/file`

Changes proposed in this pull request:

## Testing

- set up prod environment (vms or physical)
- `make build-debs`
- `scp` and install `securedrop-app-code` package
- observe no AppArmor denied messages in `/var/log/syslog` and no ossec email alerts full of apparmor denied messages.

## Deployment

New and existing installs should be covered by the updated `securedrop-app-code` deb package

## Checklist

### If you made changes to the app code:


### If you made changes to the system configuration:


### If you made changes to documentation:
